README.md

```diff
@@ -45,8 +45,8 @@ The following workflow is for a new **Helm** operator:
 - [mercurial][mercurial_tool] version 3.9+
 - [docker][docker_tool] version 17.03+.
   - Alternatively [podman][podman_tool] `v1.2.0+` or [buildah][buildah_tool] `v1.7+`
-- [kubectl][kubectl_tool] version v1.11.3+.
-- Access to a Kubernetes v1.11.3+ cluster.
+- [kubectl][kubectl_tool] version v1.15.4+.
```

Comment:
Technically SDK projects built with `operator-sdk` v0.12.0 are compatible with kube v1.12+.

This issue brings up our need for a compatibility matrix [again](https://github.com/operator-framework/operator-sdk/issues/2118).

Reply:
```suggestion
- [kubectl][kubectl_tool] version v1.12.0+.
```